packages/playwright-core/src/utils/processLauncher.ts

```diff
@@ -226,7 +226,7 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
     killSet.delete(killProcessAndCleanup);
     removeProcessHandlersIfNeeded();
     options.log(`[pid=${spawnedProcess.pid}] <kill>`);
-    if (spawnedProcess.pid && !spawnedProcess.killed && !processClosed) {
```

Comment:
Previously we wouldn't send the signal again if it has already been received, I believe it shouldn't matter for SIGKILL handling, but would be nice to double check.

Reply:
This change was needed for when we used `launchedProcess.kill()`. We don't use that anymore, so i'll see if I can remove this change.

Reply:
yup, removed in https://github.com/microsoft/playwright/pull/34130/commits/50305cc2357e832740aef2024c00eac8bb634af1